src/components/player/ModeStatsGrid.vue

```diff
@@ -89,26 +88,10 @@ export default class ModeStatsGrid extends Vue {
       sortable: false,
     },
     {
-      text: "Wins",
+      text: "Win/Loss",
       align: "end",
```

Comment:
Can you try start align. I think it will look better.

Reply:
Tried start, still doesnt look great. Center looks to be best option